packages/i18n/lib/enActionsLabel.ts

```diff
@@ -42,6 +42,8 @@ export const enActionsLabel: Record<string, Record<string, string>> = {
     timestamp: 'Message Timestamp',
     kafka_ext_header_key: 'Kafka Header Key',
     kafka_ext_header_value: 'Kafka Header Value',
+    max_linger_time: 'Max Linger Time',
+    max_linger_bytes: 'Max Linger Bytes',
```

Comment:
Move them to the `common`? Maybe Confluent needs these new configs too?
<img width="693" alt="image" src="https://github.com/user-attachments/assets/fb8810e8-5683-4d13-82ed-233ab0590e5b">